### DISCUSSION TITLE
gocryptfs reporting wrong remaining space

### DISCUSSION THREAD

**Discussion by lorenzolamasse** [rfjakob/gocryptfs#651] [Q&A]
When mounting under Monterey, disk global size is reported as being 14GB, I don't know where that value comes from (the mount source is on a Google shared folder that is not limited and the local drive has 500GB disk space), when I put files in the mounted drive, the remaining space lowers a bit (like 1/10th of what I put in it roughly), and I of course cannot put big files in it at all due to capacity.
Any idea why this happens ?

```
Filesystem                                           Size   Used  Avail Capacity   iused    ifree %iused  Mounted on
/Volumes/GoogleDrive/Shared drives/Audio/gocryptfs   14Gi  9.7Gi  4.8Gi    67% 162259398 80578145   67%   /Users/yoto/gocryptfs/music.clear
```

cheers,

**Comment by rfjakob** [comment]
Hmm, can you run

https://github.com/rfjakob/gocryptfs/files/2975356/gocryptfs.statfs-darwin.tar.gz

against the Google shared folder?

Sounds like a variant of https://github.com/rfjakob/gocryptfs/issues/375

**Comment by lorenzolamasse** [comment]
Hello,

Just tried, but I cannot mount folder at all with the version you sent me:
fuse.NewServer failed: fork/exec /Library/Filesystems/osxfusefs.fs/Support/load_osxfusefs: no such file or directory
